spec/struct.dd

```diff
@@ -566,6 +566,206 @@ $(GNAME Postblit):
+        an aggregated postblit is generated which calls `__fieldPostblit` first
+        and then `__postblit`.)
+        $(LI `void __xpostblit()`. The field and aggregated postblits, although
+        generated for a struct, are not actual struct members. In order to be able
+        to call them, the compiler internally creates an alias, called `__xpostblit`
+        which is a member of the struct and which points to the generated postblit that
+        is the most inclusive.)
+        )
+
+$(SPEC_RUNNABLE_EXAMPLE_COMPILE
+---
+// struct with alias __xpostblit = __postblit
+struct X
+{
+    this(this) {}
+}
+
+// struct with alias __xpostblit = __fieldPostblit
+// which contains a call to X.__xpostblit
+struct Y
+{
+    X a;
+}
+
+// struct with alias __xpostblit = __aggrPostblit which contains
+// a call to Y.__xpostblit and a call to Z.__postblit
+struct Z
+{
+    Y a;
+    this(this) {}
+}
+
+void main()
+{
+    // X has __postblit and __xpostblit (pointing to __postblit)
+    static assert(__traits(hasMember, X, "__postblit"));
+    static assert(__traits(hasMember, X, "__xpostblit"));
+
+    // Y does not have __postblit, but has __xpostblit (pointing to __fieldPostblit)
+    static assert(!__traits(hasMember, Y, "__postblit"));
+    static assert(__traits(hasMember, Y, "__xpostblit"));
+    // __fieldPostblit is not a member of the struct
+    static assert(!__traits(hasMember, Y, "__fieldPostblit"));
+
+    // Z has  __postblit and __xpostblit (pointing to __aggrPostblit)
+    static assert(__traits(hasMember, Z, "__postblit"));
+    static assert(__traits(hasMember, Z, "__xpostblit"));
+    // __aggrPostblit is not a member of the struct
+    static assert(!__traits(hasMember, Z, "__aggrPostblit"));
+}
+---
+)
+
+        $(P Neither of the above postblits is defined for structs that don't
+        define `this(this)` and don't have fields that transitively define it.
+        Example:)
+
+$(SPEC_RUNNABLE_EXAMPLE_COMPILE
+---
+struct X
+{}
+
+struct Y
+{
+    int a;
+    X b;
+}
+
+void main()
+{
+    static assert(!__traits(hasMember, X, "__postblit"));
+    static assert(!__traits(hasMember, X, "__xpostblit"));
+
+    static assert(!__traits(hasMember, Y, "__postblit"));
+    static assert(!__traits(hasMember, Y, "__xpostblit"));
+}
+---
+)
+
+        $(P Postblits are not overloadable. If two or more postblits are defined,
+        even if the signatures differ, the compiler assigns the
+        `__postblit` name to both and later issue a conflicting function
+        name error:)
+
+---
+struct X
+{
+    this(this) {}
+    this(this) const {} // error: function X.__postblit conflicts with function X.__postblit
+}
+---
+
+        $(P The following describes the behavior of the
+        qualified postblit definitions:)
+
+        $(OL
+        $(LI `const`. When a postblit is qualified with `const` as in
+        $(D this(this) const;) or $(D const this(this);) then the postblit
```

Comment:
We will deprecate this with the next version, but @wilzbach argued that information that gets outdated should not be in the spec. I thought that saying it's ill-formed but not mentioning the deprecation is the best compromise since you can actually call the function but it's not doing what it is supposed to.

Reply:
It's helpful to give that information so people who see invalid code getting compiled don't get confused.